File: 프로그래머스/1/86051. 없는 숫자 더하기/없는 숫자 더하기.py
'''
0~9까지 숫자 중 numbers 배열에 없는 숫자 모두 더하기

for을 사용해 10까지 돌린 후 numbers 안에 숫자와 일치 하지 않으면 더하기
'''
def solution(numbers):
    answer = 0
    for i in range(10):
        if i not in numbers:
            answer += i
    return answer

File: 프로그래머스/1/86051. 없는 숫자 더하기/없는 숫자 더하기.java
import java.util.Arrays;
class Solution {
    public int solution(int[] numbers) {
        int answer = 0;
        int[] n = new int[10];
        
        for (int i = 0; i < numbers.length; i++){
            n[numbers[i]] = 1;
        }
        
        for (int i = 0; i < n.length; i++){
            if(n[i]==0) answer += i;
        }
                
        return answer;
    }
}